common/static/coffee/spec/discussion/discussion_spec_helper.coffee

```diff
@@ -546,7 +546,7 @@ browser and pasting the output.  When that file changes, this one should be rege
         <li class="actions-item">
             <a href="javascript:void(0)" class="action-list-item action-edit" role="button">
                 <span class="action-label">Edit</span>
-                <span class="action-icon"><i class="icon fa fa-pencil"></i></span>
+                <span class="action-icon"><i class="icon fa fa-pencil-square-o"></i></span>
```

Comment:
NOTE: syncing up icons used for "editing" actions (currently just discussions + now notes) in the LMS.